aviary/subsystems/propulsion/test/test_prop_performance.py

```diff
@@ -103,6 +103,10 @@ def test_case_0_1_2(self):
         prob.set_val(Dynamic.Mission.PROPELLER_TIP_SPEED,
                      [800.0, 800.0, 750.0], units="ft/s")
```

Comment:
Can you remove all of these `PROPELLER_TIP_SPEED` `set_val` calls in this test now?

Reply:
You are right. I removed it.